config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    
+    <!-- Input files for NewlineAtEndOfFileCheckTest, intentional no new line at the end. -->
+    <suppress checks="NewlineAtEndOfFile" files="test.*Newline.*AtEndOfFile\.java"/>
```

Comment:
`test` -> `[\\/]test[\\/]`.
Append `[\\/]` to the beginning of `Newline`.
Without directory slashes, name is too general imo.

Do similar changes to other suppressions.